src/main/java/org/jenkinsci/plugins/workflow/actions/ExecutorTaskInfoAction.java

```diff
@@ -0,0 +1,88 @@
+package org.jenkinsci.plugins.workflow.actions;
+
+import hudson.model.InvisibleAction;
+import org.jenkinsci.plugins.workflow.graph.FlowNode;
+
+import javax.annotation.CheckForNull;
+import javax.annotation.Nonnull;
+
+/**
+ * Records information for a {@code node} block.
+ */
+public class ExecutorTaskInfoAction extends InvisibleAction implements PersistentAction {
```

Comment:
Ah, yeah, we could definitely remove this once we add the `WorkspaceAction`. That seems reasonable.